src/featureFlags.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Should we name this `src/FeatureFlags` since it should be a singleton? Totally not necessary, just thinking out loud. Should this file also be included in the exports in `index.js`?

Reply:
We can do either way for former. For the latter - I'm thinking of reserving feature flags for internal usage to avoid addition of backward-compatibility concerns. We can move this file to `src/internal` in that sense.